### PR TITLE
Feature/to do list

### PR DESCRIPTION
**Pull request by nattsnb**
created to do list

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
+      <div class="column-content"></div>
+    </div>
+    <div id="done-column" class="column">
+      <div class="column-title-wrapper">
+        <div class="column-title">done</div>
+      </div>
+      <div class="column-content"></div>
+    </div>
+  </div>
+</div>
```
Comment (WojciechBaczynski) [inline index.html]: please correct indentation

File: index.html [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <script src="https://kit.fontawesome.com/c1d6f06782.js" crossorigin="anonymous"></script>
 </head>
-
-<body>
-placeholder
+<body id="general-fonts">
+<div id="general-wrapper">
+  <div id="input-section">
+    <div id="input-wrapper">
```
Comment (WojciechBaczynski) [inline index.html:14]: why 2 divs instead of one?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        addEventListenersToToDoSection(taskWrapper)
+    })
+}
+
+function addEventListenersToToDoSection(taskBox) {
+    const deleteIconFunction = taskBox.querySelector(".delete-icon");
+    if (deleteIconFunction) {
+        deleteIconFunction.addEventListener("click", function(){
+            taskBox.remove();
+    })
```
Comment (WojciechBaczynski) [inline script.js]: please correct indentation

File: script.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    }
+    const doingColumn = document.querySelector("#doing-column")
```
Comment (WojciechBaczynski) [inline script.js]: semicolon

File: style.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+#add-icon {
+    padding-left: 8px;
+    font-size: 28px;
+    font-weight: bold;
+}
+
+#column-section {
+    display: flex;
+    justify-content: space-evenly;
+    padding: 0 10px 0 10px;
```
Comment (WojciechBaczynski) [inline style.css:35]: it can be written in a shorter way :)

File: script.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
+const taskInput = document.querySelector("#input-box");
+const addIcon = document.querySelector("#add-icon");
+const toDoColumn = document.querySelector("#to-do-column")
+const doingColumn = document.querySelector("#doing-column")
+const doneColumn = document.querySelector("#done-column")
```
Comment (WojciechBaczynski) [inline script.js]: semicolons

File: script.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+        leftIcon.classList.add("icon-task", "left-icon", "fas", "fa-arrow-alt-circle-left");
+        const deleteIcon = document.createElement("i");
+        deleteIcon.classList.add("icon-task", "delete-icon", "fas", "fa-trash-alt");
+        const rightIcon = document.createElement("i");
+        rightIcon.classList.add("icon-task", "right-icon", "fas", "fa-arrow-alt-circle-right");
+        taskWrapper.append(taskName)
+        taskWrapper.append(taskIconBar)
+        taskIconBar.append(leftIcon, deleteIcon, rightIcon)
+        toDoColumn.append(taskWrapper)
+        addEventListenersToToDoSection(taskWrapper)
```
Comment (WojciechBaczynski) [inline script.js]: semicolons

File: style.css [hunk 8]
```diff
@@ -0,0 +1,101 @@
+#general-fonts {
+    font-family: Arial, sans-serif;
+}
+
+#general-wrapper {
+    width: 800px;
+    height: 634px;
+    background-color: #F2F2F2;
+}
```
Comment (WojciechBaczynski) [inline style.css]: why you use id for this general styles? for font family for example you could set it on * {} or something like that

File: index.html (hunk 2 above)
Comment (nattsnb) [reply index.html:14]: i couldn't align the input box and button properly without another div

File: style.css (hunk 5 above)
Comment (nattsnb) [reply style.css:35]: no idea, tell me please

**Pull request merged**